server/server.py

```diff
@@ -50,6 +54,10 @@ def handler404(e):
 
 requests.packages.urllib3.disable_warnings(requests.packages.urllib3.exceptions.InsecureRequestWarning)
 
+class NetworkIDsToName(Enum):
```

Comment:
Similarly here - many of these helper functions (mapping networks to FCs, reversing enum name to its key definition) could best be in their own file and imported as needed.

Reply:
Moved them to another file